### ISSUE TITLE
T&C for museum display

### ISSUE THREAD

**Issue by hyanwong** [OneZoom/OZtree#93]
Separately to the normal T&C, we should describe the pros / cons of showing wikipedia to potential users of the museum display. E.g. you have all the species etc wikipedia pages in multiple languages, but we have gone some way to mitigating abuse by ensuring that users can't just display any web page on the screen (or indeed any wikipedia page). However, we can't guarantee the appropriateness of the wikipedia pages (NB: we should also make a museum display with WP turned off - this is trivial). For example, the species "[Wonga wonga](http://beta.onezoom.org/life.html/@Amaryllidinae=5764812?pop=ol_5092180#x3960,y-922,w14.2990)" has no english language WP page, so online versions show information about the payday loan company, with the text "We have no record of an exact wikipedia page. The page below is the first page returned by searching en.wikipedia.org by name for ‘Wonga wonga’". 

Also we can't stop someone editing the WP page to include inappropriate material, then going to that page on the display (this seems unlikely, though). Trivial hacks of this sort could be partially mitigated by looking at the revisions of a page and not using revisions within e.g. the past hour, but this might be overkill for us.

**Comment by jrosindell** [comment]
I agree with this and when we release a museum display we will have a launch page with some instructions and presumably a pointer to T&C which will be expanded to encompass MD terms

**Comment by hyanwong** [comment]
This is a similar issue to https://github.com/OneZoom/OZtree/issues/53

**Comment by hyanwong** [comment]
Do we have this now @jrosindell ? I've lost track!

**Comment by jrosindell** [comment]
https://www.onezoom.org/education/museum_display_setup.html

"All links that could enable a user to click and get out of the page are removed, including from the Wikipedia information page insets. This should make it impossible for your visitors to click on anything in the page that might result in displaying inappropriate content. "

add "However please note that we cannot guarantee the content on Wikipedia that users might be able to access through the display."
